utils/rowgroup/rowgroup.h

```diff
@@ -143,8 +147,12 @@ class StringStore
   StringStore();
   virtual ~StringStore();
 
-  inline std::string getString(uint64_t offset) const;
-  uint64_t storeString(const uint8_t* data, uint32_t length);  // returns the offset
+  inline utils::NullString getString(uint64_t offset) const;
+  // returns the offset.
+  // it may receive nullptr as data and it is proper way to store NULL values.
+  uint64_t storeString(const uint8_t* data, uint32_t length);
+  //uint64_t storeString(const utils::NullString& str);  // returns the offset
```

Comment:
We don't need these comments.

Reply:
I think you are talking about commented out storeString. If this is the case, yes, we do not need this comment.

Three other comment lines are valuable in my opinion.